ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Genesis/Governor.hs

```diff
@@ -127,6 +128,30 @@ reprocessLoEBlocksOnCandidateChange chainDb getCandidates =
         reprocessLoEAsync chainDb
         spin newTips
 
+runLoEUpdaterOnChange ::
+  (Monoid a, Eq a, IOLike m, LedgerSupportsProtocol blk) =>
+  UpdateLoEFrag m blk ->
+  m ( UpdateLoEFrag m blk
+    , ChainDB m blk ->
+      STM m a -> -- Re-run the LoE updater whenever this changes
+      m Void
```

Comment:
Document what the returned computation does.

Also, I don't understand what the return `UpdateLoEFrag m blk` is for.

Reply:
This can probably be refactored if we remove `UpdateLoEFrag` as brought up by @tek (also see [Slack](https://moduscreate.slack.com/archives/C05CEUFFH7Z/p1710783577790569)).

Reply:
I untangled it a bit